react-payments-app/src/components/Input/CardPassword/input.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect, useRef } from "react";
+import { MAX_INPUT_LENGTH } from "../../../common/constant";
+
+const CardPasswordInput = ({ onChange, handleSubmit }) => {
+  const [password, setPassword] = useState(["", ""]);
+  const [error, setError] = useState("");
+  const nextElement = useRef(null);
+  const [event, setEvent] = useState(null);
+
+  const setPasswordByInput = (index) => (e) => {
+    const updatedPassword = [...password];
+    const { value } = e.target;
+
+    if (Number.isNaN(+value)) {
+      setError("숫자만 입력해주세요.");
+      return;
+    }
+
+    setEvent(e);
+    updatedPassword[index] = value;
+    setPassword(updatedPassword);
+
+    if (index === 0) {
+      nextElement.current.focus();
+    }
+  };
+
+  useEffect(() => {
+    onChange(password, error);
+    if (password[1]) {
+      console.log(password[1]);
+      handleSubmit(event);
+    }
```

Comment:
```suggestion
    if (password[1]) {
      handleSubmit(event);
    }
```


매우 많은 입력값을 유저로부터 받았는데, 유저가 확인할 시간도 없이, 바로 넘어가는건 좀 어색한 UX인거 같아요.

피그마에 있는 `다음` 버튼을 구현해보면 어떨까요?